example/src/plugins/job-ui-single/pages/JobForm.tsx

```diff
@@ -98,11 +97,15 @@ export const JobForm = (props: {
                   </p>
                 </div>
                 <EntityPickerButton
-                  returnLinkReference={true}
-                  onChange={(linkReferenceEntity: TLinkReference) => {
+                  onChange={(address: string, entity: TValidEntity) => {
+                    const ref: TLinkReference = {
```

Comment:
I generally prefer full names over abbreviations :) 

```suggestion
                    const linkReference: TLinkReference = {
```

Reply:
I generally agree, though I find abbreviations useful when the scope is small, ie `[1,2,3].map((x) => x*x)`. But this scope can easily be extended, and then an abbreviation would be bad. Will fix